mitiq/interface/mitiq_qiskit/conversions.py

```diff
@@ -247,6 +247,7 @@ def _transform_registers(
     circuit._qubit_set = set()
     circuit.qregs = []
     circuit._data = []
+    circuit._qubit_indices = {}
```

Comment:
Just checking my understanding, before this change inconsistencies could arise in the converted circuits because the "old" qubit indices were carried through the conversion?

Reply:
Yes, exactly. Moreover, without resetting `_qubit_indeces` the function `_transform_registers()` sometimes failed to add some new qubits as expected.